index.html

```diff
@@ -91,7 +91,7 @@ <h2>Docs</h2>
           <h2>Downloads</h2>
           <p>Download tarballs or zipfiles straight from the version control
              tags, so you can start working with LTP straight away.</p>
-          <p><a class="btn" href="https://github.com/linux-test-project/ltp/tags">View downloads <i class="icon-tags"></i></a></p>
+          <p><a class="btn" href="https://github.com/linux-test-project/ltp/releases">View downloads <i class="icon-tags"></i></a></p>
```

Comment:
I think that the tags view is actually better than the releases, since the table is easier to read.

The other two changes looks good.

Reply:
OK, I'll merge it without this change.